rootfs/etc/services.d/supervisor/finish

```diff
@@ -1,5 +1,8 @@
-#!/usr/bin/execlineb -S0
+#!/usr/bin/execlineb -S1
 # ==============================================================================
 # Take down the S6 supervision tree when Supervisor fails
 # ==============================================================================
+if { s6-test ${1} -ne 0 }
```

Comment:
This prevents the OTA updater to close itself for systemd to create a new container from updated image

Reply:
Right, because it closes down with a 0 exit code. 👍 
Will adjust.

Reply:
Ok, so since the loop is stopped nicely, it will return exit 0.
Which is nice, but problematic here, as we can't tell the difference if the program is shut down by itself or something else.

The s6-svscan might be shutting down already, causing the error we see. However, this error doesn't make a difference. As we are either shutting down already OR killing it because of a crash. In both cases, the supervision tree needs to be taken down.

As there is no direct way to test the current state of the svscan, simply suppressing the error, in this case, will do. The result is the same either way.

Reply:
why not just `-eq 0`?

Reply:
however, I'm fine with that

Reply:
Because it can still be a nice exit, causing the svscan already being terminated/shutting down (raising the same error).